docs/faq.rst

```diff
@@ -64,7 +64,7 @@ Does Scrapy work with HTTP proxies?
 
 Yes. Support for HTTP proxies is provided (since Scrapy 0.8) through the HTTP
 Proxy downloader middleware. See
-:class:`~scrapy.contrib.downloadermiddleware.httpproxy.HttpProxyMiddleware`.
+:class:`~scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware`.
```

Comment:
I forgot, it's used with a trailing 's' in settings too, we have `DOWNLOADER_MIDDLEWARES` and `SPIDER_MIDDLEWARES`. I think I prefer to keep it as "middlewares", seems like its usage is consistent in all places where is referred.